src/sbs/instruments/acs.py

```diff
@@ -0,0 +1,194 @@
+from datetime import datetime
+import numpy as np
+import re
+import xarray as xr
+
+from src.sbs.core import NUM_PAT
+
+class Dev():
```

Comment:
We'll probably want to rename this. DevParser or ACSFileParser come to mind but I'll follow up with something concrete